aten/src/ATen/xpu/XPUEvent.h

```diff
@@ -156,6 +156,20 @@ struct TORCH_XPU_API XPUEvent {
   }
 
  private:
+  void assignEvent(sycl::queue& queue) {
+    if (enable_timing_) {
+      event_ = std::make_unique<sycl::event>(
+          sycl::ext::oneapi::experimental::submit_profiling_tag(queue));
```

Comment:
Do we know when they will move this API from experimental namespace?

Reply:
I assume it should be a long-term plan. Let me confirm with the compiler team.

Reply:
The compiler team has a plan to move them from experimental since 2025.1